src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -1999,6 +2088,14 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         }
 
         _UpdateSystemParameterSettings();
+
+        // Only update the appearance here if an unfocused config exists -

```

Comment:
Oh we'll probably just reset the settings in the normal `UpdateSettings` path, so this is fine

Reply:
Thanks for catching this! We set the control's unfocused appearance to null if the new settings doesn't define one (in case it used to define one but no longer does)